src/lib/services/audit-discovery.ts

```
@@ -968,11 +1015,18 @@ export class AuditDiscoveryService {
 
       // Deduplicate audits at the method level as well
       const deduplicatedAudits = this.deduplicateAudits(audits)
-      console.log(`🔍 Scraped ${url}: ${audits.length} → ${deduplicatedAudits.length} unique audits`)
+      } catch (linkError) {
+      }
+      console.log(`🔍 Scraped ${normalizedUrl} (original: ${url}): ${audits.length} → ${deduplicatedAudits.length} unique audits`)
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)